lib/result-view.js

```diff
@@ -190,7 +190,9 @@ export default class ResultView {
     //   });
     // });
     const div = document.createElement("div");
-    ReactDOM.render(<Transform data={bundle.get(mimeType)} />, div);
+    if (mimeType) {
```

Comment:
I think we should move the check and  above [this line](https://github.com/nteract/hydrogen/pull/746/files#diff-375c469d3548707f41002f462a83207fR178) and completely return from the function if there's no supported mime type. What do you think?